### PR TITLE
Added VMware check

### PR DESCRIPTION
**Pull request by frapposelli**
Signed-off-by: Fabio Rapposelli fabio@vmware.com

### CODE REVIEW

File: rootfs/rootfs/etc/rc.d/vmtoolsd [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-if [ ! -d /mnt/hgfs ]; then
-	mkdir -p /mnt/hgfs
-	# load shared folders kernel module
-	modprobe vmhgfs
-	# try to mount the root shared folder, this command can fail
-	# if shared folders are disabled on the host, vmtoolsd will take care of the mount
-	# if they are enabled while the machine is running.
-	mount -t vmhgfs .host:/ /mnt/hgfs
-fi
+SYSTYPE=$(cat /sys/class/dmi/id/sys_vendor | grep -i vmware | wc -l)
```
Comment (tianon) [inline rootfs/rootfs/etc/rc.d/vmtoolsd]: Why not just `grep -ic vmware` instead of piping to `wc`?

Comment (frapposelli) [inline rootfs/rootfs/etc/rc.d/vmtoolsd]: d'oh, it's just my brain that is wired that way :smile: fixed now.

Comment (tianon) [inline rootfs/rootfs/etc/rc.d/vmtoolsd]: "ic" :+1: (sorry, I definitely didn't plan that pun the second I typed out
my original comment...)

**Comment by tianon** [comment]
So, the thing this doesn't answer is why we need this in the first place.  Did something change about `vmtoolsd`?

**Comment by frapposelli** [comment]
vmtoolsd doesn't create the mount point, and will not mount it if shared folders are already enabled in the VM, this script is a very short substitute of the original script that comes with vmware-tools.

**Comment by tianon** [comment]
Ahh, ok.  Silly "vmtoolsd". :+1:

**Comment by SvenDowideit** [comment]
much yay :)

**Pull request merged**